src/promptflow/promptflow/_sdk/operations/_test_submitter.py

```diff
@@ -216,57 +204,57 @@ def change_logger_level(level):
             yield
             logger.setLevel(origin_level)
 
-        @contextlib.contextmanager
-        def add_prefix():
-            write = sys.stdout.write
-
-            def prefix_output(*args, **kwargs):
-                if args[0].strip():
-                    write(f"{Fore.LIGHTBLUE_EX}[{self.dataplane_flow.name}]: ")
-                write(*args, **kwargs)
-
-            sys.stdout.write = prefix_output
-            yield
-            sys.stdout.write = write
+        def show_node_log_and_output(node_run_infos, show_node_output):
+            """Show stdout and output of nodes."""
+            for node_name, node_result in node_run_infos.items():
+                # Prefix of node stdout is "%Y-%m-%dT%H:%M:%S%z"
+                pattern = r"\[\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\+\d{4}\] "
+                node_logs = re.sub(pattern, "", node_result.logs["stdout"])
+                if node_logs:
+                    for log in node_logs.rstrip("\n").split("\n"):
+                        print(f"{Fore.LIGHTBLUE_EX}[{node_name}]:", end=" ")
```

Comment:
This may behave differently in different shells, please double confirm.

Reply:
especially output to bash/powershell/notebook.

Reply:
https://github.com/tartley/colorama
colorma is used to cross-platform colored terminal text in Python

Reply:
@lalala123123 have we tested in those shells? Seems they are supported according to colorama's doc
![image](https://github.com/microsoft/promptflow/assets/7776147/7f453669-d033-43d6-8a06-e5bcf24b7027)